### ISSUE TITLE
Compiler errors

### ISSUE THREAD

**Issue by bestdan** [aryzhov/flutter-expandable#34]
I'm using AndroidStudio, just hit this problem: 

1. Added `expandable ^4.0.0` to pubspec.yaml (no issue)
2. Imported (not yet called) --> 

```
Launching lib/main.dart on iPhone 11 Pro Max in debug mode...

Compiler message:
../../.pub-cache/hosted/pub.dartlang.org/expandable-4.0.0/lib/expandable.dart:147:19: Error: The method 'dependOnInheritedWidgetOfExactType' isn't defined for the class 'BuildContext'.
 - 'BuildContext' is from 'package:flutter/src/widgets/framework.dart' ('/usr/local/Cellar/flutter/stable/packages/flutter/lib/src/widgets/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'dependOnInheritedWidgetOfExactType'.
        ? context.dependOnInheritedWidgetOfExactType<_ExpandableThemeNotifier>()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dartlang.org/expandable-4.0.0/lib/expandable.dart:148:19: Error: The method 'findAncestorWidgetOfExactType' isn't defined for the class 'BuildContext'.
 - 'BuildContext' is from 'package:flutter/src/widgets/framework.dart' ('/usr/local/Cellar/flutter/stable/packages/flutter/lib/src/widgets/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'findAncestorWidgetOfExactType'.
        : context.findAncestorWidgetOfExactType<_ExpandableThemeNotifier>();
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dartlang.org/expandable-4.0.0/lib/expandable.dart:169:42: Error: The method 'dependOnInheritedWidgetOfExactType' isn't defined for the class 'BuildContext'.
 - 'BuildContext' is from 'package:flutter/src/widgets/framework.dart' ('/usr/local/Cellar/flutter/stable/packages/flutter/lib/src/widgets/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'dependOnInheritedWidgetOfExactType'.
    _ExpandableThemeNotifier n = context.dependOnInheritedWidgetOfExactType<_ExpandableThemeNotifier>();
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dartlang.org/expandable-4.0.0/lib/expandable.dart:284:19: Error: The method 'dependOnInheritedWidgetOfExactType' isn't defined for the class 'BuildContext'.
 - 'BuildContext' is from 'package:flutter/src/widgets/framework.dart' ('/usr/local/Cellar/flutter/stable/packages/flutter/lib/src/widgets/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'dependOnInheritedWidgetOfExactType'.
        ? context.dependOnInheritedWidgetOfExactType<_ExpandableControllerNotifier>()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dartlang.org/expandable-4.0.0/lib/expandable.dart:285:19: Error: The method 'findAncestorWidgetOfExactType' isn't defined for the class 'BuildContext'.
 - 'BuildContext' is from 'package:flutter/src/widgets/framework.dart' ('/usr/local/Cellar/flutter/stable/packages/flutter/lib/src/widgets/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'findAncestorWidgetOfExactType'.
        : context.findAncestorWidgetOfExactType<_ExpandableControllerNotifier>();
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Compiler failed on /flutter_project/lib/main.dart
Error launching application on iPhone 11 Pro Max.
```

**Comment by aryzhov** [comment]
Thank you for pointing that out. I was working off the dev branch. Will
change to the supported API in stable.

**Comment by aryzhov** [comment]
I have investigated the issue. I switched to the stable branch of Flutter, and the package and the example app compiles fine. My Flutter version is 1.12.13+hotfix.5. I suspect your version of Flutter is different. I suggest two options for you: 'flutter upgrade' or using version 3.0 of expandable: `expandable: "^3.0.0"`. In order to avoid future confusion, I updated the pubspec to require Flutter version 1.12 and released expandable version 4.0.1.
